### PR TITLE
update dependencies.

### PR DESCRIPTION
**Pull request by martinyunify**
update logrus and adopt glide.

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package: github.com/yunify/qingcloud-sdk-go
+import:
+- package: github.com/DATA-DOG/godog
```
Comment (pengsrc) [inline glide.yaml]: Test dependency should be put in a separated glide file, and the `vendor` directory can be ignored.

Refer: https://github.com/yunify/qingstor-sdk-go/tree/master/test

Comment (jolestar) [inline glide.yaml]: or use testImport

Comment (martinyunify) [inline glide.yaml]: I would like to take some time to refactor the test case after I resolve cni work item.

Comment (martinyunify) [inline glide.yaml]: use go test to run godoc command and update build scripts

**Pull request merged**